src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx

```diff
@@ -525,6 +525,9 @@
   <data name="CustomUrlPathCleaned.Error" xml:space="preserve">
     <value>The Page URL entered contains characters which cannot be used in a URL or are illegal characters for a URL.   These characters have been removed.  Click the Update button again to accept the modified URL.&lt;br&gt;[NOTE: The illegal characters list is the following: &lt;&gt;\?:&amp;=+|%# ]</value>
   </data>
+  <data name="CustomUrlPortalAlias.Error" xml:space="preserve">
+    <value>Please, select a valid site alias for this custom URL.</value>
```

Comment:
There should no comma following **Please** 🎩 🇬🇧

Reply:
typo